pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/FieldMergeRule.scala

```diff
@@ -0,0 +1,77 @@
+package uk.ac.wellcome.platform.merger.rules
+import uk.ac.wellcome.models.work.internal.{
+  TransformedBaseWork,
+  UnidentifiedWork
+}
+import uk.ac.wellcome.platform.merger.rules.WorkFilters.WorkFilter
+
+case class MergeResult[T](field: T, redirects: Seq[TransformedBaseWork])
```

Comment:
Would be helpful having a comment documenting this type. Also I think `data` (or something like that) would be a better name than `field` (see my other comment)